code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -178,11 +178,12 @@
 
 	activated = TRUE
 	linked_hive.check_if_hit_larva_from_pylon_limit()
-	addtimer(CALLBACK(src, PROC_REF(give_larva)), XENO_PYLON_ACTIVATION_COOLDOWN, TIMER_UNIQUE|TIMER_OVERRIDE|TIMER_LOOP|TIMER_DELETE_ME)
+	addtimer(CALLBACK(src, PROC_REF(give_larva)), XENO_PYLON_ACTIVATION_COOLDOWN, TIMER_UNIQUE|TIMER_OVERRIDE|TIMER_LOOP|TIMER_DELETE_ME|TIMER_NO_HASH_WAIT)
```

Comment:
Unnecessary, but doesn't hurt either. You only need to TIMER_NO_HASH_WAIT if the duration changes, and in this case it can't change.